hoverxref-intersphinx.rst

```diff
@@ -0,0 +1,114 @@
+.. post:: June 10, 2021
+   :tags: czi, grant, czi-grant, embed, sphinx-hoverxref
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      New release of sphinx-hoverxref, our Sphinx extension to add tooltips to cross references,
+      with support for intersphinx.
+
+New release of sphinx-hoverxref with support for intersphinx
+============================================================
+
+We have released version 0.6b1 of `sphinx-hoverxref`_,
+a Sphinx extension that shows a floating window
+that embeds the content of a cross-reference.
+
+This extension is an essential part
+of the work we are doing to improve interoperability of documentation in general,
+and scientific documentation in particular,
+thanks to the `CZI grant </czi-grant-announcement>`_ we received last year.
+
+.. figure:: img/sphinx-hoverxref-math.png
+   :width: 80%
+   :align: center
+   :alt: sphinx-hoverxref displaying a tooltip including an equation
+   :target: _images/sphinx-hoverxref-math.png
+
+   sphinx-hoverxref displaying a tooltip including an equation
+
+.. _sphinx-hoverxref: https://sphinx-hoverxref.readthedocs.io/
+
+Embedding documentation fragments
```

Comment:
This is another place where `previews` is probably more descriptive. Talk about the _value_ to the user, not the technical implementation. Nobody cares about fragments, or even knows what it means in this context. 

```suggestion
Embedding reference previews
```

Not 100% sold on this, but feels better.

Reply:
What about "content preview"?

Reply:
(I went ahead and renamed it to "content preview", let me know if you still prefer "reference preview" 👍🏽)

Reply:
That works for me.